debug/process/stateExport.go

```diff
@@ -0,0 +1,169 @@
+package process
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go/common"
+	"github.com/ElrondNetwork/elrond-go/state"
+)
+
+// ExportUserAccountState will try to export the account state of a provided address
+func ExportUserAccountState(accountsDB state.AccountsAdapter, identifier string, address []byte, parentDirForFiles string) {
```

Comment:
might return error? otherwise, remove the `try` from the comment

Reply:
no, it is meant to easily usable when modifying the code ad-hoc